services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return new_movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    sessions = MovieSession.objects.all()
+
+    if session_date:
+        sessions = sessions.filter(show_time__year=session_date.split("-")[0],
+                                   show_time__month=session_date.split("-")[1],
+                                   show_time__day=session_date.split("-")[2])
```

Comment:
The `session_date` parsing assumes the date format is always 'YYYY-MM-DD'. Consider adding validation to ensure the date string is in the correct format to prevent runtime errors.